src/freenet/l10n/freenet.l10n.en.properties

```diff
@@ -541,7 +541,7 @@ FirstTimeWizardToadlet.continue=Continue
 FirstTimeWizardToadlet.currentPrefix=Current size:
 FirstTimeWizardToadlet.currentSpeed=Current limit:
 FirstTimeWizardToadlet.datastoreSize=Datastore size
-FirstTimeWizardToadlet.datastoreSizeLong=Please select a size for your datastore. The datastore acts like a cache; storing data for the network will help you to get better throughput when requesting popular files. The more space you can afford the better it is for the community and the faster your node and especially your downloads will go.
+FirstTimeWizardToadlet.datastoreSizeLong=Please select a size for your datastore. Recommended values lie between 1GiB and 20GiB. A smaller datastore can result in problems downloading files, a larger datastore is beneficial to the network but might slow down your computer due to high filesystem load, especially in the first few hours after installation. The datastore acts like a cache; storing data for the network will help you to get better throughput when requesting popular files. The more space you can afford the better it is for the community and the faster your node and especially your downloads will go.
```

Comment:
s/, a larger/, and a larger/

I'm also wondering about removing some of the additional elaboration as overly verbose.